### PR TITLE
Remove mention of the module search_api_solr_defaults module, fixes #7

### LINKED ISSUE #7: README needs to be updated, since search_api_solr_defaults module no longer exists
It appears the search_api_solr_defaults module was removed from search_api_solr 4.x
https://www.drupal.org/project/search_api_solr/issues/3295726

The step in the README can't be used.

"Enable the Search API Solr Search Defaults module: ddev drush en -y search_api_solr_defaults. (If it can't be enabled due to the "article" content type not existing, you can just create a search_api_solr server manually.)"

### PR DESCRIPTION
**Pull request by cspitzlay**
That module does not exist anymore (and was only meant for demonstration purposes).

### CODE REVIEW

**Comment by rfay** [comment]
Thanks so much! Did you manually go through the setup on a fresh project using the new instructions?

**Comment by cspitzlay** [comment]
No, I'm afraid I did not. But I got the admin path from my working project and verified that the config from the former search_api_solr_defaults module did use the standard connector.  The actual settings to get it to work are ddev-specific, so I did not change them.

**Comment by rfay** [comment]
It's easy to test out in a generic d9 project, hope a couple of people with solr experience can do that. Thanks!

File: README.md [hunk 1]
```diff
@@ -8,11 +8,12 @@ This repository allows you to quickly install Apache Solr for Drupal 9 into a [D
 
 1. `ddev get drud/ddev-drupal9-solr && ddev restart`
 1. You may need to install the relevant Drupal requirements: `ddev composer require drush/drush:* drupal/search_api_solr`
```
Comment (tyler36) [inline README.md:10]: We should also include the `drupal/search_api` module here, otherwise the URL in the next step is a 404.

Comment (tyler36) [reply README.md:10]: Oh, I see. It's a dependency. However the steps don't explicitly state to 'enable the module'.
IIRC, there was a recipe that required config before enabling the module (?) so I did not enable it yet.

File: README.md [hunk 2]
```diff
@@ -8,11 +8,12 @@ This repository allows you to quickly install Apache Solr for Drupal 9 into a [D
...
-1. Enable the Search API Solr Search Defaults module: `ddev drush en -y search_api_solr_defaults`. (If it can't be enabled due to the "article" content type not existing, you can just create a search_api_solr server manually.)
-1. Edit the enabled search_api server named `default_solr_server` at `admin/config/search/search-api/server/default_solr_server/edit`
-   * set "Solr host" to `solr`
-   * set "Solr core" name to "dev"
-   * Under "Advanced server configuration" set the "solr.install.dir" to `/opt/solr`
+1. Create a search_api server at `admin/config/search/search-api` -> "Add server"
+1. Choose Solr as backend.
```
Comment (tyler36) [inline README.md:13]: The form would not submit until I provided a "Server Name" and "Machine-readable name"

**Comment by tyler36** [comment]
After following these instruction, I recieve the following error banner:

```
Status message
The server was successfully saved.
```

```
Error message
You are using outdated Solr configuration set. Please follow the instructions described in the [README.md](https://solr-d94-demo.ddev.site/modules/contrib/search_api_solr/README.md) file for setting up Solr.
```

File: README.md (hunk 1 above)
Comment (rfay) [inline README.md:10]:
```suggestion
1. You may need to install the relevant Drupal requirements: `ddev composer require drush/drush:* drupal/search_api_solr`
1. Enable the `search_api_solr` module either using the web interface or `ddev drush en -y search_api_solr`
```

**Comment by rfay** [comment]
It seems that the complaint on configuring the solr core "You are using outdated Solr configuration set" is just complaining that we're using the [classic way](https://git.drupalcode.org/project/search_api_solr/-/blob/4.x/README.md#setting-up-solr-single-core-the-classic-way). I don't know why it shows as an error. Am I right that that's the meaning of the error @mklabrenner?

ddev___Umami_Food_Magazine

File: README.md [hunk 3]
```diff
@@ -8,11 +8,12 @@ This repository allows you to quickly install Apache Solr for Drupal 9 into a [D
...
+1. Choose "Standard" as the Solr connector type and configure it:
```
Comment (rfay) [inline README.md:14]:
```suggestion
1. Choose "Standard" as the Solr connector type and configure it:
    * Set "Server name" to anything you want. Maybe "ddev-solr-server".
    * Set "Backend" to "Solr"
```

**Review by rfay — CHANGES_REQUESTED** [review]
It's coming along, thanks!

Please add to the "What is this" that this strategy is based on the [Setting up Solr (single core) - the classic way](https://git.drupalcode.org/project/search_api_solr/-/blob/4.x/README.md#setting-up-solr-single-core-the-classic-way) in the search_api_solr README.md

File: README.md (hunk 1 above)
Comment (cspitzlay) [reply README.md:10]: Thanks. I just committed the README change suggested by @rfay about that.

**Comment by cspitzlay** [comment]
> Please add to the "What is this" that this strategy is based on the [Setting up Solr (single core) - the classic way](https://git.drupalcode.org/project/search_api_solr/-/blob/4.x/README.md#setting-up-solr-single-core-the-classic-way) in the search_api_solr README.md

Done.

File: README.md (hunk 2 above)
Comment (cspitzlay) [reply README.md:13]: I accepted @rfay 's suggested changes.

**Review by rfay — APPROVED** [review]
I went through the whole process again and it worked great. Thanks for all the work on this @cspitzlay  and for your review @tyler36

**Pull request merged**